### DISCUSSION TITLE
[EPUB] Add search

### DISCUSSION THREAD

**Discussion by Shohreh** [sumatrapdfreader/sumatrapdf#3581] [Ideas]
Hello,

Unlike PDFs, EPUBs, can't be searched. I have to use STDUViewer, which I like less than SumatraPDF.

Cheers,

**Comment by GitHubRulesOK** [comment]
You must be using a prior version
Recent versions are totally different so keep the old one installed until converted, I suggest download a newer as [Portable version: Only](https://www.sumatrapdfreader.org/download-free-pdf-viewer)

With the historic two interfaces
to use search you would need to switch to fixed layout so in setting need `UseFixedPageUI = true`
but it will alter appearance too see https://github.com/sumatrapdfreader/sumatrapdf/issues/284#issuecomment-581414778

**Comment by Shohreh** [comment]
Indeed. When I saw "3.3.3" on [this page](https://www.sumatrapdfreader.org/free-pdf-reader), I thought no newer version was available.

Search is available in the latest and greatest.
